### PR TITLE
Note about type='text/css' in Fx75

### PR DESCRIPTION
**Pull request by jmswisher**
- [Bugzilla bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1614329)
- [Sprints issue](https://github.com/mdn/sprints/issues/2923)

### CODE REVIEW

File: html/elements/style.json [hunk 1]
```diff
@@ -265,7 +265,8 @@
                 "version_added": "12"
               },
               "firefox": {
-                "version_added": "1"
+                "version_added": "1",
+                "notes": "Prior to 75, Firefox accepted any media (MIME) type. Starting in 75, Firefox accepts only 'text/css', per the spec."
```
Comment (evilpie) [inline html/elements/style.json]: I think this is not really correct. Prior Firefox version parsed this as a MIME type string, so stuff like `text/css; charset=utf-8` would be allowed (additionally also the empty string?). Not arbitrary MIME types.

Comment (jmswisher) [reply html/elements/style.json]: @evilpie How would you suggest rephrasing? "Prior to 75, Firefox parsed this as a media (MIME) type string."?

Comment (evilpie) [reply html/elements/style.json]: Maybe "Prior to 75, Firefox allowed any CSS MIME type with optional parameters, this has been restricted to the literal string `text/css` per spec" ?

Comment (jmswisher) [reply html/elements/style.json]: OK, thanks.

**Comment by mont1985** [comment]
Atylo escriba

**Pull request merged**